users/config.rst

```diff
@@ -980,6 +1173,75 @@ setLowPriority
     disable this behavior, for example to control process priority yourself
     as part of launching Syncthing, set this option to ``false``.
 
+.. option:: options.maxFolderConcurrency
+
+    This option controls how many folders may concurrently be in I/O-intensive
+    operations such as syncing or scanning.  The mechanism is described in
+    detail in a :doc:`separate chapter </advanced/option-max-concurrency>`.
+
+.. option:: options.crashReportingURL
+
+    Server URL where :doc:`automatic crash reports <crashrep>` will be sent if
+    enabled.
+
+.. option:: options.crashReportingEnabled
+
+    Switch to opt out from the :doc:`automatic crash reporting <crashrep>`
+    feature to avoid Syncthing "phoning home" on serious troubles.  Defaults to
+    ``true``, to help the developers troubleshoot.
+
+.. option:: options.databaseTuning
+
+    Controls how Syncthing uses the backend key-value database that stores the
+    index data and other persistent data it needs.  The available options and
+    implications are explained in a :doc:`separate chapter
+    </advanced/option-database-tuning>`.
+
+.. option:: options.maxConcurrentIncomingRequestKiB
+
+    This limits how many bytes we have "in the air" in the form of response data
+    being read and processed.
+
+.. option:: options.announceLANAddresses
+
+    Enable (the default) or disable announcing private (RFC1918) LAN IP
+    addresses to global discovery.
+
+.. option:: options.sendFullIndexOnUpgrade
+
+    Controls whether all index data is resent when an upgrade is detected,
```

Comment:
```suggestion
    Controls whether all index data is resent when an upgrade has happened,
```

Reply:
Is that an upgrade of the local Syncthing executable?  If so, your suggestion makes sense.  Though if it also includes upgrades remote devices, then "detect" is probably appropriate.

Reply:
It is the local executable. I had this option enabled for a few months, and I also remember the same behaviour when it always took place after upgrading Syncthing.

Technically, it doesn't even have to be an "upgrade". Simply replacing the executable with a different one (e.g. a custom-built or an older version) will trigger the index exchange.